build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -37,14 +39,11 @@ public class CompileTargets
             { "debian.8-x64", "debian.8-x64" }
         };
 
-
-        private static string DotnetHostBaseName => $"dotnet{Constants.ExeSuffix}";
-        private static string DotnetHostFxrBaseName => $"{Constants.DynamicLibPrefix}hostfxr{Constants.DynamicLibSuffix}";
-        private static string HostPolicyBaseName => $"{Constants.DynamicLibPrefix}hostpolicy{Constants.DynamicLibSuffix}";
-
         [Target(nameof(PrepareTargets.Init), 
             nameof(CompileCoreHost), 
-            nameof(PackagePkgProjects))]
+            nameof(PackagePkgProjects),
+            nameof(RestoreLockedCoreHost),
```

Comment:
this relates back to our discussion yesterday on the two-step process. This isolates the decision of which version of the host should be used to the sharedfx portion of the build.